LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org>
+Copyright (c) 2015-2016 Vincent Hanquez <vincent@snarc.org> 2016-2017
```

Comment:
I don't really have a link for that, I derive this information from the legal department of a previous big company TM. Most things I'm finding on the internet seems either confused, outdated, or just claim something different, so sadly no certainty here.

copyright is valid regardless of the copyright notice anyway, it's just easier to claim "accidental violation of copyright" if the code is not littered with copyright notice.